process/process.go

```diff
@@ -209,6 +218,33 @@ func loadProcess(ctx context.Context, pid int) (*Process, error) {
 	}
 	process.ParentPid = int(ppid)
 
+	// Get all environment variables
+	envVariables, err := pInfo.EnvironWithContext(ctx)
+
+	// Not important enough to stop execution of the function
+	if err == nil {
+		// Try to find PORTMASTER_PROFILE variable
+		for _, envVar := range envVariables {
+			if strings.HasPrefix(envVar, profileEnvironmentVariable) {
+				// Found. Check validity and set to process
+				splitted := strings.SplitN(envVar, "=", 2)
```

Comment:
It might be easier to just use `strings.TrimPrefix()` if we already know what it is.